django/core/management/utils.py

```diff
@@ -10,15 +10,16 @@
 from .base import CommandError
 
 
-def popen_wrapper(args, os_err_exc_type=CommandError, universal_newlines=True):
+def popen_wrapper(args, os_err_exc_type=CommandError, stdout_encoding='utf-8'):
     """
     Friendly wrapper around Popen.
 
     Returns stdout output, stderr output and OS status code.
     """
     try:
+        # Using universal_newlines may produce unexpected and annoying decoding errors
```

Comment:
universal_newlines=False seems to be the default, do we need to be explicit about it? If so, please wrap comment at 79 chars and add period.